plugins/linux-pipewire/linux-pipewire.c

```diff
@@ -45,7 +45,7 @@ bool obs_module_load(void)
 		pipewire_capture_load();
 		break;
 	case OBS_NIX_PLATFORM_X11_GLX:
-		break;
+		return false;
```

Comment:
I think the entire `switch` statement here can be removed and just call `pipewire_capture_load()`. We can reduce 10 lines.